core/selection.js

```diff
@@ -1168,7 +1168,8 @@
 					range = sel.createRange();
 
 				// The selection range is reported on host, but actually it should applies to the content doc.
-				if ( sel.type != 'None' && range.parentElement().ownerDocument == doc.$ )
+				// #9780 added check if range.parentElement() exists
```

Comment:
Remember to put ticket number at the end of the comment like:
> The `parentElement` may be null for read only mode in IE10 and below. (#9780)

Also try to describe the case why this change was introduced (like above).